pytest/tests/mocknet/load_testing_helper.py

```diff
@@ -52,7 +47,9 @@ def get_nonce_for_pk(account_id, pk, finality='optimistic'):
             "account_id": account_id,
             "finality": finality
         })
+    assert len(access_keys['result']['keys']) > 0
     for k in access_keys['result']['keys']:
+        assert 'public_key' in k
```

Comment:
KeyError will be thrown before if this is not true so perhaps this is unnecessary?